Program.cs

```diff
@@ -3,17 +3,57 @@
 // Copyright (c) Metamation India.                                                
 // ---------------------------------------------------------------------------
 // Program.cs                                                                     
-// 
-// 
-// 
+// Program to swap the indices of a given list of numbers
+// A random series of numbers is displayed to the user and the user is asked to provide the two indices to be swapped.
+// The number series after swapping the indices is displayed.
 // ---------------------------------------------------------------------------------------
-using static System.Console;
+
 namespace Training {
    #region class Program ------------------------------------------------------------------------
+   /// <summary>Swap Indices</summary>
    internal class Program {
       #region Method ----------------------------------------------
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+      /// <summary>This method prints a series of numbers based on the count given by the user and displays the updated series after swapping the user asked indices</summary>
+      static void Main () {
+         Console.WriteLine ("Enter the count (2-10) of random numbers to be displayed: ");
+         int count = ValidInputCheck ();
+         if (count < 2) Console.WriteLine ("You atleast need two numbers to swap indices.");

```

Comment:
You have added a check for count less than 2, but why not for count > 10 as expected range is 2 to 10?